firmware/Sources/APP/APP_SALG/app_salg.c

```diff
@@ -43,6 +44,42 @@
 /**********************************************************************************/
 volatile uint8_t n_ints= 0, int_triggered = 0;
 
+
+/*		LIMITS  	*/
+MID_REG_limit_s APP_SALG_limit = {};
+
+MID_REG_control_s APP_SALG_consign = {
+		MID_REG_DISABLED, 	// outStatus
+		MID_REG_MODE_IDLE, 	// mode
+		MID_REG_LIMIT_TIME, // limitType
+		0,					// modeRef
+		0					// limRef
+};
+MID_REG_control_s APP_SALG_control = {
+		MID_REG_DISABLED, 	// outStatus
+		MID_REG_MODE_IDLE, 	// mode
+		MID_REG_LIMIT_TIME, // limitType
+		0,					// modeRef
+		0					// limRef
+};
+MID_REG_errorStatus_s APP_SALG_errorStatus = {
+		MID_REG_ERROR_NONE,	// hsVoltErr
+		MID_REG_ERROR_NONE,	// lsVoltErr
+		MID_REG_ERROR_NONE,	// lsCurrErr
+		MID_REG_ERROR_NONE,	// commErr
+		MID_REG_ERROR_NONE,	// tempErr
+		MID_REG_ERROR_NONE,	// intErr
+		0					// lastErrVal
+};
+MID_REG_meas_s APP_SALG_meas = {
```

Comment:
Change to MID_REG_data_s and use it for meas, factors and offset on EPC for calibration

Reply:
Meas are used in several modules meanwhile, factors and offsets are only used in MID DABS. Wouldn't it be better is they are in separated structs?

Reply:
Understood. I changed MID_REG_meas_s to MID_REG_meas_property_s to create instances of the same type in EPC_CONF with the calibration values. :)